source/Bell-Logging/StandardErrorStructuredLogger.class.st

```diff
@@ -6,6 +6,8 @@ Class {
 
 { #category : #accessing }
 StandardErrorStructuredLogger class >> instance [
+	"Avoid using GsFile>>stderr for debugging purposes
+	 (returns nil when connected via GemBuilder on VAST64 for Windows)"
```

Comment:
Is this comment still valid? It will not return nil anymore

Reply:
Not valid. Fixed in 63dc3e2f5044b5f0ed36c294557b1c2dc69207fe